packages/typespec-ts/tsconfig.test.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "module": "ESNext",
-    "moduleResolution": "Node"
+    "module": "node16",
+    "moduleResolution": "node16"
```

Comment:
This is now getting into some conflict where the generated tests are not compatible with node16 imports